### DISCUSSION TITLE
Bug on not able to download a LoRA

### DISCUSSION THREAD

**Discussion by TomLucidor** [LykosAI/StabilityMatrix#279] [Q&A] [answered]
Currently trying to install some LoRAs but that sometimes this error appears
![image](https://github.com/LykosAI/StabilityMatrix/assets/85554801/8d1c4b47-dc87-4c31-9a62-e18415cc5871)

**Comment by tscolin** [comment]
Yup, civitai requires a login to download some content (per the creator).  Im not sure how to login from matrix.

**Reply by TomLucidor** [reply]
There has to be a cookie hack somewhere that gets this to work

**Reply by TomLucidor** [reply, marked as answer]
Nope they really are enforcing this in the newer versions, looks like things aren't gonna get easier

**Reply by mohnjiles** [reply]
You can login to civitai by providing an API Key in Settings -> Accounts -> CivitAI -> Connect.

**Reply by TomLucidor** [reply]
@mohnjiles in hindsight I did add the API Key, still slightly salty they had this new rule
